js/playgame.js

```diff
@@ -0,0 +1,39 @@
+// This code is based on Chris Marriott's Unicorn game found here:
```

Comment:
I take it that PlayGame is responsible for all game logistics involving scoring, time, and any housekeeping outside of actually playing the game?